src/main/java/org/folio/rest/utils/Mapper.java

```diff
@@ -0,0 +1,9 @@
+package org.folio.rest.utils;
+
+import org.w3c.dom.Node;
+
+import java.io.IOException;
+
+public interface Mapper {
```

Comment:
I would propose to add javadocs at least for new interfaces/classes/enums (and its primary public methods) to give an idea of what the component is.

Reply:
Added javadocs.